applications/API/filters.py

```
@@ -0,0 +1,19 @@
+from django_filters import rest_framework as filters
+
+from .models import Insult
+
+
+class InsultFilter(filters.FilterSet):
+    class Meta:
+        model = Insult
+        fields = {
+            "nsfw": ["exact"],
+            "category": ["exact"],
+        }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)